packages/manager/src/index.ts

```diff
@@ -13,7 +13,7 @@ export { createSliceMachineManager } from "./managers/createSliceMachineManager"
 export { createSliceMachineManagerMiddleware } from "./managers/createSliceMachineManagerMiddleware";
 export type { CreateSliceMachineManagerMiddlewareArgs } from "./managers/createSliceMachineManagerMiddleware";
 
-export type { Environment } from "./managers/prismicRepository/types";
+export type { Environment, Limit } from "./managers/prismicRepository/types";
```

Comment:
Could we use a more descriptive name than `Limit`, such as `PushChangesLimit`?

I know this type already existed before this PR, but now that it is part of the public API, its name should be self-describing.

Reply:
Thanks I agree!

Reply:
Done